spec/tapioca/compilers/dsl/active_record_fixtures_spec.rb

```diff
@@ -0,0 +1,168 @@
+        class Post < ActiveRecord::Base
+        end
+      RUBY
+
+      expected = <<~RBI
+        # typed: strong
+      RBI
+
+      assert_equal(expected, rbi_for("ActiveSupport::TestCase"))
+    end
+
+    it("generates methods for fixtures") do
+      add_ruby_file("schema.rb", <<~RUBY)
+        ActiveRecord::Migration.suppress_messages do
+          ActiveRecord::Schema.define do
+            create_table :posts do |t|
+              t.string :title
+              t.string :author
+              t.timestamps
+            end
+          end
+        end
+      RUBY
+
+      add_ruby_file("post.rb", <<~RUBY)
+        class Post < ActiveRecord::Base
+        end
+      RUBY
+
+      add_content_file("test/fixtures/posts.yml", <<~YAML)
+        super_post:
+          title: An incredible Ruby post
+          author: Johnny Developer
+          created_at: 2021-09-08 11:00:00
+          updated_at: 2021-09-08 11:00:00
+      YAML
+
+      expected = <<~RBI
+        # typed: strong
+
+        class ActiveSupport::TestCase
+          sig { params(fixture_names: Symbol).returns(Post) }
+          def posts(*fixture_names); end
+        end
+      RBI
+
+      assert_equal(expected, rbi_for("ActiveSupport::TestCase"))
+    end
+
+    it("generates methods for fixtures from multiple sources") do
+      add_ruby_file("schema.rb", <<~RUBY)
+        ActiveRecord::Migration.suppress_messages do
+          ActiveRecord::Schema.define do
+            create_table :posts do |t|
+              t.string :title
+              t.string :author
+              t.timestamps
+            end
+
+            create_table :users do |t|
+              t.string :first_name
+              t.string :last_name
+              t.timestamps
+            end
+          end
+        end
+      RUBY
+
+      add_ruby_file("post.rb", <<~RUBY)
+        class Post < ActiveRecord::Base
+        end
+      RUBY
+
+      add_ruby_file("user.rb", <<~RUBY)
+        class User < ActiveRecord::Base
+        end
+      RUBY
+
+      add_content_file("test/fixtures/posts.yml", <<~YAML)
+        super_post:
+          title: An incredible Ruby post
+          author: Johnny Developer
+          created_at: 2021-09-08 11:00:00
+          updated_at: 2021-09-08 11:00:00
+      YAML
+
+      add_content_file("test/fixtures/users.yml", <<~YAML)
+        customer:
+          first_name: John
+          last_name: Doe
+          created_at: 2021-09-08 11:00:00
+          updated_at: 2021-09-08 11:00:00
+      YAML
+
+      expected = <<~RBI
+        # typed: strong
+
+        class ActiveSupport::TestCase
+          sig { params(fixture_names: Symbol).returns(Post) }
+          def posts(*fixture_names); end
+
+          sig { params(fixture_names: Symbol).returns(User) }
+          def users(*fixture_names); end
+        end
+      RBI
+
+      assert_equal(expected, rbi_for("ActiveSupport::TestCase"))
+    end
+  end
+
+  private
+
+  sig { void }
+  def fake_rails_app
```

Comment:
Can we rename this method to be `define_fake_rails_app` or something, to make it clear what it is doing?